Annex60/Fluid/Interfaces/ConservationEquation.mo

```diff
@@ -236,7 +239,7 @@ equation
   if traceDynamics == Modelica.Fluid.Types.Dynamics.SteadyState then
     zeros(Medium.nC)  = mbC_flow;
```

Comment:
@Mathadon : I will add C_flow to the steady-state equation because what is added from the connector has to leave from the ports.